mass_mailing_custom_unsubscribe/views/mail_unsubscription_view.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data>
+
+<record id="mail_unsubscription_view_form" model="ir.ui.view">
+    <field name="name">Mail Unsubscription Form</field>
+    <field name="model">mail.unsubscription</field>
+    <field name="arch" type="xml">
+        <form>
+            <sheet>
+                <group>
+                    <field name="date"/>
+                    <field name="mass_mailing_id"/>
+                    <field name="unsubscriber_id"/>
+                    <field name="email"/>
+                    <field name="success"/>
+                    <field name="reason_id"/>
+                    <field
```

Comment:
I tend to care less about parsers and more about humans, when we speak about whitespaces. Since the contents are Python, I wrote them as such, but nevermind I'll change it.